src/net-to-gross/calcInsurance.js

```diff
@@ -0,0 +1,47 @@
+const Big = require('big.js');
+const BIG_20 = new Big(20);
+const findRegionAndPolicy = require('../findPolicyAndRegion');
+
+module.exports = function calcInsurance(net, totalTax, region = 1, date = new Date()) {
+    return new Promise(function (resolve, reject) {
+        const netBig = new Big(net);
+        const totalNetAndTotalTax = netBig.add(totalTax);
+
+        const info = findRegionAndPolicy(region, date)
+        const minWage = info.minWage;
```

Comment:
it is not necessary to declare `minWage` and `baseSalary` here because info has them